src/test/scala/mesosphere/marathon/tasks/ResourceUtilTest.scala

```diff
@@ -15,26 +18,48 @@ class ResourceUtilTest extends FunSuite with GivenWhenThen with Assertions {
 
   test("resource mix") {
     val leftOvers = ResourceUtil.consumeResources(
-      Seq(scalar("cpus", 3), ports("ports", 2 to 20), set("labels", Set("a", "b"))),
-      Seq(scalar("cpus", 2), ports("ports", 2 to 12), set("labels", Set("a")))
+      Seq(MTH.scalarResource("cpus", 3), ports("ports", 2 to 20), set("labels", Set("a", "b"))),
+      Seq(MTH.scalarResource("cpus", 2), ports("ports", 2 to 12), set("labels", Set("a")))
     )
-    assert(leftOvers == Seq(scalar("cpus", 1), ports("ports", 13 to 20), set("labels", Set("b"))))
+    assert(leftOvers == Seq(MTH.scalarResource("cpus", 1), ports("ports", 13 to 20), set("labels", Set("b"))))
   }
 
   test("resource repeated consumed resources with the same name/role") {
     val leftOvers = ResourceUtil.consumeResources(
-      Seq(scalar("cpus", 3)),
-      Seq(scalar("cpus", 2), scalar("cpus", 1))
+      Seq(MTH.scalarResource("cpus", 3)),
+      Seq(MTH.scalarResource("cpus", 2), MTH.scalarResource("cpus", 1))
     )
     assert(leftOvers == Seq())
   }
 
   test("resource consumption considers roles") {
     val leftOvers = ResourceUtil.consumeResources(
-      Seq(scalar("cpus", 2), scalar("cpus", 2, role = "marathon")),
-      Seq(scalar("cpus", 0.5), scalar("cpus", 1, role = "marathon"), scalar("cpus", 0.5, role = "marathon"))
+      Seq(MTH.scalarResource("cpus", 2), MTH.scalarResource("cpus", 2, role = "marathon")),
+      Seq(MTH.scalarResource("cpus", 0.5), MTH.scalarResource("cpus", 1, role = "marathon"), MTH.scalarResource("cpus", 0.5, role = "marathon"))
     )
-    assert(leftOvers == Seq(scalar("cpus", 1.5), scalar("cpus", 0.5, role = "marathon")))
+    assert(leftOvers == Seq(MTH.scalarResource("cpus", 1.5), MTH.scalarResource("cpus", 0.5, role = "marathon")))
+  }
+
+  test("resource consumption considers reservation") {
```

Comment:
The testName does not really reflect what is tested here – shouldn't the test code call `ResourceUtil.consumeResources`?